eng/Build.props

```diff
@@ -23,8 +23,25 @@
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Bcl.AsyncInterfaces.6.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Encodings.Web.6.0.0.csproj" />
     <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\System.Text.Json.6.0.0.csproj" />
-
+    
+    <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Build.Framework.17.3.2.csproj" />
```

Comment:
These have to be defined in the correct dependency order in order to prevent prebuilts.  e.g. System.Security.Permissions.6.0.0 must be built before Microsoft.Build.Framework.17.3.2.

Reply:
Should be ordered correctly now